infrastructure/template.yaml

```diff
@@ -417,6 +417,27 @@ Resources:
           MetricNamespace: !Sub "${AWS::StackName}/LogMessages"
           MetricName: "OTGFunction-Fatalerror"
 
+  CheckHmrcLambdaConcurrency80Alarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub ${AWS::StackName}-CheckHmrcLambdaConcurrency80-alarm
+      AlarmDescription: !Sub Check HMRC ${Environment} lambdas alarm if over 80% of concurrent executions is used ${SupportManualURL}"
+      ActionsEnabled: true
+      AlarmActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      OKActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      InsufficientDataActions: []
+      MetricName: ConcurrentExecutions
+      Namespace: AWS/Lambda
+      Statistic: Maximum
+      Period: 60
+      EvaluationPeriods: 1
+      DatapointsToAlarm: 1
+      Threshold: 780
```

Comment:
Would you be able to elaborate on where the number 780 came from?

Reply:
We have 975 across all our environments up-to-prod.

Reply:
oh wow so if we add/remove lambdas we have to remember to adjust this number?

Reply:
Misread that the first time - ok what determines all of our provisioned concurrency? is 975 a default?

Reply:
SREs - but yes we use defaults. The account limit is 1000 but we have 975 available.